CRM/Contact/Selector.php

```diff
@@ -492,7 +492,16 @@ public function &getColumnHeaders($action = NULL, $output = NULL) {
           if (trim($phoneType) && !is_numeric($phoneType) && strtolower($phoneType) != $fld) {
             $title .= "-{$phoneType}";
           }
-          $title .= " ($loc)";
+          // fetch Location type label from name as $loc, which will be later used in column header
```

Comment:
@eileenmcnaughton added a comment here :)

Reply:
Is there a reason why you removed the whitespace at the beginning?